### PR TITLE
Story/KS-155/current position blur

### PR DESCRIPTION
**Pull request by DanaDovzhenko**
**Jira ticket:**
https://protopie.atlassian.net/browse/KS-155

Light theme:
image

Dark theme:
image

### CODE REVIEW

File: src/react-guides/Guides.tsx [hunk 1]
```diff
@@ -199,7 +199,7 @@ export default class Guides extends React.PureComponent<GuidesProps, GuidesState
   private dragPositionElement() {
     const { className } = this.props as Required<GuidesProps>;
     return (
-      this.props.displayDragPos && <div className={`${prefix('wrapper-pos')} ${DISPLAY_DRAG} ${prefix(className)}`} ref={ref(this, 'displayElement')} style={this.props.dragGuideStyle} />
+      this.props.displayDragPos && <div className={`${prefix('wrapper-pos')} ${prefix(className)}`} ref={ref(this, 'displayElement')} style={this.props.dragGuideStyle} />
```
Comment (nickbohdanov) [inline src/react-guides/Guides.tsx]: probably the line is to long

Comment (nickbohdanov) [reply src/react-guides/Guides.tsx]: and not readable

**Review by nickbohdanov — CHANGES_REQUESTED** [review]

File: src/react-guides/consts.ts [hunk 2]
```diff
@@ -87,11 +87,11 @@ canvas {
 }
 
 .light {
-    background: linear-gradient(90deg, rgba(255,255,255,0) 0%, rgba(255,255,255,0.6) 15%, rgba(255,255,255,1) 35%, rgba(255,255,255,1) 50%, rgba(255,255,255,1) 65%, rgba(255,255,255,0.6) 85%, rgba(255,255,255,0) 100%);
+    background: linear-gradient(90deg, rgba(255,255,255,0) 0%, rgba(255,255,255,0.4) 15%, rgba(255,255,255,0.4) 35%, rgba(255,255,255,0.4) 50%, rgba(255,255,255,0.4) 65%, rgba(255,255,255,0.4) 85%, rgba(255,255,255,0) 100%);
 }
 
 .dark {
-    background: linear-gradient(90deg, rgba(68,68,64,0) 0%, rgba(68,68,68,0.6) 15%, rgba(68,68,68,1) 35%, rgba(68,68,68,1) 50%, rgba(68,68,68,1) 65%, rgba(68,68,68,0.6) 85%, rgba(68,68,68,0) 100%);
+    background: linear-gradient(90deg, rgba(68,68,64,0) 0%, rgba(68,68,68,0.6) 15%, rgba(68,68,68,0.9) 35%, rgba(68,68,68,0.9) 50%, rgba(68,68,68,0.9) 65%, rgba(68,68,68,0.6) 85%, rgba(68,68,68,0) 100%);
```
Comment (nickbohdanov) [inline src/react-guides/consts.ts]: I think it exceed max line length

**Review by nickbohdanov — CHANGES_REQUESTED** [review]

File: src/react-guides/consts.ts (hunk 2 above)
Comment (DanaDovzhenko) [reply src/react-guides/consts.ts]: @nickbohdanov  Yes, thank you, I fixed it

File: src/react-guides/Guides.tsx [hunk 3]
```diff
@@ -491,7 +491,7 @@ export default class Guides extends React.PureComponent<GuidesProps, GuidesState
   private showDragPosition(nextPos: number, guidePos: number) {
     if (this.props.displayDragPos) {
       const translate = this.isHorizontal ? this.calcHorizontalTransform(nextPos) : this.calcVerticalTransform(nextPos);
-      this.displayElement.style.cssText += 'display: block; transform: ' + translate;
+      this.displayElement.style.cssText += 'display: flex; transform: ' + translate;
```
Comment (nickbohdanov) [inline src/react-guides/Guides.tsx]: Use template string
```suggestion
      this.displayElement.style.cssText += `display: flex; transform: ${translate}`;
```

**Review by nickbohdanov — CHANGES_REQUESTED** [review]
Looks good to me, I only have one minor suggestion.

File: src/react-guides/Guides.tsx (hunk 3 above)
Comment (DanaDovzhenko) [reply src/react-guides/Guides.tsx]: Thank you

File: src/react-guides/Guides.tsx [hunk 4]
```diff
@@ -200,10 +200,10 @@ export default class Guides extends React.PureComponent<GuidesProps, GuidesState
     const { className } = this.props as Required<GuidesProps>;
     return (
       this.props.displayDragPos && (
-        <div 
-          className={`${prefix('wrapper-pos')} ${DISPLAY_DRAG} ${prefix(className)}`} 
-          ref={ref(this, 'displayElement')} 
-          style={this.props.dragGuideStyle} 
+        <div
+          className={`${prefix('wrapper-pos', className)}`}
```
Comment (nickbohdanov) [inline src/react-guides/Guides.tsx]: You can write like this.
In such case you can delete extra variable classname and use it like props directly
```suggestion
          className={`${prefix('wrapper-pos', this.props.className)}`}
```

**Review by nickbohdanov — CHANGES_REQUESTED** [review]
Seems like the last comment ;)

File: src/react-guides/Guides.tsx [hunk 5]
```diff
@@ -200,10 +200,10 @@ export default class Guides extends React.PureComponent<GuidesProps, GuidesState
     const { className } = this.props as Required<GuidesProps>;
```
Comment (nickbohdanov) [inline src/react-guides/Guides.tsx]: can be removed
probably with type

**Review by nickbohdanov — APPROVED** [review]

**Pull request merged**